.pre-commit-config.yaml

```diff
@@ -12,6 +12,7 @@ repos:
         entry: poetry run isort
         language: system
         types: [file, python]
+        args: ["--profile", "black", "--filter-files"]
```

Comment:
I spotted we've swapped for Ruff elsewhere. I'm happy with either, but we should probably standardize to one :)

Reply:
I really like Ruff, I format/lint on save with it's VSCode extension. It implements the same rules as Black and iSort, so it's mostly a drop in replacement, but bundles them into one tool and works much faster.

Reply:
This is the pre-commit I'm using: https://github.com/mapaction/cod-ab-data-quality/blob/78194d374353a03967743e3611af0be5bba791f5/.pre-commit-config.yaml